homeassistant/components/verisure.py

```diff
@@ -150,20 +147,25 @@ def reconnect(self):
         if not self._lock.acquire(blocking=False):
             return
         try:
+            if self._password_retries < 0:
+                _LOGGER.error('Wrong password for Verisure, update config')
+                self._reconnect_timeout = time.time() + 60*60*24
```

Comment:
I wanted to repeat the error message, but not too often. As long as the password isn't updated in the config would be risk to retry since the user might be locked out. 

I could change so that the error is only printed once, if you think it is redundant. 

Would be nice to be able to set a device in a failed state that was visible in the UI.